db/migrate/20221018121447_add_user_and_post_ref_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserAndPostRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly reference the `author` instead of `user` based on the feedback above

Reply:
@julie-ify